mapping_plugin/slam_toolbox_rviz_plugin.cpp

```diff
@@ -199,6 +212,18 @@ void SlamToolboxPlugin::LoadPoseGraph()
   }
 }
 
+/*****************************************************************************/
+  void SlamToolboxPlugin::LoadMap()
```

Comment:
function named load map, called in submap, but not? This needs to be cleaned up and separated. There are 2 executables.
Merging 
main slam

These need to be easily distinguished and kept completely separately.